tests/integration/test_send_crash_reports/fake_sentry_server.py

```diff
@@ -1,25 +1,26 @@
-import BaseHTTPServer
+import http.server
 
 RESULT_PATH = '/result.txt'
 
 
-class SentryHandler(BaseHTTPServer.BaseHTTPRequestHandler):
+class SentryHandler(http.server.BaseHTTPRequestHandler):
     def do_POST(self):
         post_data = self.__read_and_decode_post_data()
         with open(RESULT_PATH, 'w') as f:
+            content_length = self.headers.get("content-length")
             if self.headers.get("content-type") != "application/x-sentry-envelope":
                 f.write("INCORRECT_CONTENT_TYPE")
-            elif self.headers.get("content-length") < 3000:
-                f.write("INCORRECT_CONTENT_LENGTH")
-            elif '"http://6f33034cfe684dd7a3ab9875e57b1c8d@localhost:9500/5226277"' not in post_data:
+            elif int(content_length) < 200:
```

Comment:
Why is 3000 changed to 200?

Reply:
Because the payload is not that huge, and the test was always broken, since it compares `str` with number:

```python
$ python2 -c "print('263' > 3000)"
True
```

And it is fine in python2 but not in python3:

```python
$ python3 -c "print('263' > 3000)"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
TypeError: '>' not supported between instances of 'str' and 'int'
```